frontend/src/components/home_page/FeaturedProjectCard.tsx

```diff
@@ -21,19 +21,21 @@ const FeaturedProjectCard = ({ link, title, date, summary, image, altText }: Fea
         </a>
       </div>
       <div id={styles.cardTextContainer}>
-        <div id={styles.cardHeadingContainer}>
-          <div id={styles.cardTitleContainer}>
-            <a id={styles.cardTitle} href={link}>
-              <h3>{title}</h3>
-            </a>
+        <div id={styles.cardVariableTextContainer}>

```

Comment:
Note that having two classes `cardTextContainer` and `cardVariableTextContainer` might be a source of future confusion. it would probably be better to opt for something that is more descriptive like "cardDetails" and have the outer one be "cardDetailsContainer"

Reply:
@heylookitsme You're right that sounds like a better name. I can create a separate PR for that change.